README.md

```diff
@@ -105,6 +105,9 @@ CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/cri-containerd.soc
 ```
 ### Test
 See [here](./docs/testing.md) for information about test.
+## Using the crictl Client
+See [here](./docs/crictl.md) for information about using `crictl` to debug
+pods, containers, and images.
```

Comment:
/s/, and/ and/

Reply:
The comma is optional, but is correct as well.